core/src/test/java/apoc/schema/SchemasTest.java

```diff
@@ -585,4 +634,25 @@ public void testConstraintsRelationshipsAndExcludeRelationshipsValuatedShouldFai
         }
 
     }
+
+    @Test
+    public void testIndexesWithMultipleLabels() {
```

Comment:
would be good to add a test for indices with multi rel types

Reply:
Yes the name was confusing, now I've changed it.
There is already a multi-relType index, `CALL db.index.fulltext.createRelationshipIndex ('fullIdxRel', ['TYPE_1', 'TYPE_2'], ['alpha', 'beta'])`.
To maintain the same behavior as before, `apoc.schema.nodes` shows both indexes and constraints,
while `apoc.schema.relationships` only the constraints,
so in this test-case it must return an empty list, without errors unlike before (effectively this check was missing, I added it on line `658`)